scripts/build_component_container.sh

```diff
@@ -23,14 +24,22 @@ if make -C "${executable_src_path}" --no-print-directory --dry-run container >/d
 then
     make -C "${executable_src_path}" --no-print-directory --quiet container CONTAINER_REPO="${CONTAINER_REPO}" DRACON_VERSION="${DRACON_VERSION}"
 else
-    dockerfile_template="
-        FROM ${BASE_IMAGE:-scratch}                     \n
-        COPY ${executable_path} /app/${executable_path} \n
-        ENTRYPOINT ["/app/${executable_path}"]          \n
-    "
+    dockerfile_template=""
+    if [[ -v "${BASE_IMAGE+x}" && -n "${BASE_IMAGE+x}" ]]
```

Comment:
On my machine this fails with error `conditional binary operator expected`

I think this can simplified by just using `if [[ -n "${BASE_IMAGE+x}" ]]` as `-n` checks already for `BASE_IMAGE` to be set.

Works on my machine like this!